frontend/public/components/operator-lifecycle-manager/descriptors/spec/index.tsx

```diff
@@ -56,7 +57,7 @@ const BooleanSwitch: React.FC<SpecCapabilityProps> = (props) => {
 
   return <div className="co-spec-descriptor--switch">
     <AsyncComponent
-      loader={() => import('patternfly-react').then(m => m.Switch)}
+      loader={() => Switch}
```

Comment:
No reason to use `AsyncComponent` any longer if we're already importing `Switch`